packages/common/src/converter/payload-converter.ts

```diff
@@ -140,3 +153,157 @@ export class BinaryPayloadConverter extends AsyncFacadePayloadConverter {
     return content.data as any;
   }
 }
+
+abstract class ProtobufPayloadConverter extends AsyncFacadePayloadConverter {
+  protected readonly typesByName: Record<string, ProtobufCodec> = {};
```

Comment:
Do we even need this anymore? Why not just use root directly?

Reply:
Ack, thanks 😄 Was in the `protobufClasses` mindset and just looking at properties in the console and didn't think about it having methods like `lookupType()`.